packages/hoppscotch-common/src/components/collections/graphql/index.vue

```diff
@@ -11,7 +11,7 @@
         type="search"
         autocomplete="off"
         :placeholder="t('action.search')"
-        class="!border-0 bg-transparent py-2 pl-4 pr-2"
+        class="flex w-full bg-transparent px-4 py-2 h-8"
```

Comment:
Could you elaborate on why this was required? I noticed the element spanned a height of `32px` already.

Reply:
Not on safari, hence our desktop app (using safari engine to render) as well.
Hence I added a hard-coded the height.